doc/classes/TreeItem.xml

```diff
@@ -430,7 +437,7 @@
 			<return type="void" />
 			<param index="0" name="child" type="TreeItem" />
 			<description>
-				Removes the given child [TreeItem] and all its children from the [Tree]. Note that it doesn't free the item from memory, so it can be reused later. To completely remove a [TreeItem] use [method Object.free].
+				Removes the given child [TreeItem] and all its children from the [Tree]. Note that it doesn't free the item from memory, so it can be reused later (see [method add_child]). To completely remove a [TreeItem] use [method Object.free].
```

Comment:
Maybe we can cite the `move_before` and `move_after` methods here, too?

Reply:
This wasn't changed. Is there a reason for that?

Reply:
I added a line below that one.

Reply:
Oh yeah, I didn't see that.